python/jupytercad_lab/jupytercad_lab/notebook/cad_document.py

```diff
@@ -166,32 +165,6 @@ def remove_annotation(self, annotation_id: str) -> None:
             with self.ydoc.transaction() as t:
                 self._metadata.pop(t, annotation_id, None)
 
-    def set_color(self, object_name: str, color: Optional[List]) -> None:
```

Comment:
We need to re-add this method using the new `color` field of the shared model

Reply:
Ok let's do that separately. This should be resolved before final 3.0.0